beets/util/artresizer.py

```diff
@@ -66,7 +70,7 @@ class LocalBackendNotAvailableError(Exception):
 
 class LocalBackend:
     @classmethod
-    def available(cls):
+    def available(cls) -> bool:
         try:
             cls.version()
```

Comment:
We should have an abstract method `LocalBackend.version()` for this line to pass the type check.

Reply:
Is that in scope for this typing PR or a comment on something that needs to be done at another time?

Reply:
Up to you, but since this just means adding a

```python
def version(cls) -> ...:
    raise NotImplementedError()
```

I think it's not such a big change that would make this PR intractable. (Actually making `LocalBackend` an `abc.ABC` and using the `@abstractmethod` decorator would be some more work, but I guess with the above stopgap solution we already get quite far, and the `@abstractmethod` way would require it anyway.)